pageserver/src/tenant/timeline.rs

```diff
@@ -204,6 +151,8 @@ pub struct Timeline {
 
     pub(crate) layers: tokio::sync::RwLock<(LayerMap, LayerMapping)>,
 
+    pub(super) lcache: LayerCache,
```

Comment:
Feels like usually the naming in the codebase is non-abbreviating, like `layer_cache`, but I don't really feel strongly about it. However, since it does have a very long scope (duration of a Timeline) maybe it should be unabbreviated?

Reply:
renamed in 113a4256d41a3cbc9cdfb143749934edee1398de